weave/trace/weave_client.py

```diff
@@ -797,7 +797,7 @@ def add_costs(self, costs: Dict[str, CostCreateInput]) -> CostCreateRes:
             ```
 
         Args:
-            costs: Dictionary of costs to add to the project. In the form of {llm_id: cost}.
+            costs: Dictionary of costs to add to the project. In the form of `{llm_id: cost}`.
```

Comment:
this seems not quite descriptive enough. `cost` here is actually another dictionary, right? Side note: is that type checked if someone does something incorrect?